src/vs/workbench/contrib/customEditor/common/contributedCustomEditors.ts

```diff
@@ -12,15 +12,12 @@ import { IStorageService, StorageScope, StorageTarget } from 'vs/platform/storag
 import { Memento } from 'vs/workbench/common/memento';
 import { CustomEditorDescriptor, CustomEditorInfo, CustomEditorPriority } from 'vs/workbench/contrib/customEditor/common/customEditor';
 import { customEditorsExtensionPoint, ICustomEditorsExtensionPoint } from 'vs/workbench/contrib/customEditor/common/extensionPoint';
-import { DEFAULT_EDITOR_ID } from 'vs/workbench/services/editor/common/editorOpenWith';
 import { IExtensionPointUser } from 'vs/workbench/services/extensions/common/extensionsRegistry';
 
-const builtinProviderDisplayName = nls.localize('builtinProviderDisplayName', "Built-in");
-
 export const defaultCustomEditor = new CustomEditorInfo({
-	id: DEFAULT_EDITOR_ID,
+	id: 'default',
 	displayName: nls.localize('promptOpenWith.defaultEditor.displayName', "Text Editor"),
-	providerDisplayName: builtinProviderDisplayName,
+	providerDisplayName: nls.localize('builtinProviderDisplayName', "Built-in"),
```

Comment:
[nit] Try extracting this string since it it used in two places

Reply:
That is probably on me, I wasn't sure about the benefit of sharing NLS strings here. If we have a concept of built-in custom editors, shouldn't we rather have a flag to indicate as such and then maybe put the label in one central place?